.github/workflows/deploy-market.yaml

```diff
@@ -36,8 +36,10 @@ jobs:
       LINEASCAN_KEY: ${{ secrets.LINEASCAN_KEY }}
     steps:
       - name: Seacrest
-        uses: hayesgm/seacrest@v1
+        uses: hayesgm/seacrest@c3eef919c8ac21cf077aad0513ad8a493885ce6d
```

Comment:
Should we update these to @v2 now that it's out?

Reply:
Yeah changed to @v2 now :)